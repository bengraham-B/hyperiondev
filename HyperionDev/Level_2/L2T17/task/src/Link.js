import React from 'react'

export default function Link() {
  return (
    <div>
        <a href="	https://swapi.dev/api/people/1/">Star Wars API: Luke Skywalker</a>
    </div>
  )
}
