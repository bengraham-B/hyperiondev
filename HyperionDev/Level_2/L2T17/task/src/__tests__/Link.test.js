import renderer from 'react-test-renderer'
import Link from '../Link';

it('Link renders Correctly', () => {
    const tree = renderer
    .create(<Link/>)
    .toJSON()
    expect(tree).toMatchSnapshot()
})






