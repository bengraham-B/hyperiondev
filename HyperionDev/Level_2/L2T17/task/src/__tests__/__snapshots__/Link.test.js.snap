// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders Correctly 1`] = `
<div>
  <a
    href="	https://swapi.dev/api/people/1/"
  >
    Star Wars API: Luke Skywalker
  </a>
</div>
`;
